app/services/stripe_service/api/stripe_api_wrapper.rb

```diff
@@ -264,6 +264,17 @@ def update_account(community:, account_id:, attrs:)
       end
     end
 
+    def update_account_capabilities(community:, account_id:)
+      with_stripe_payment_config(community) do |payment_settings|
```

Comment:
@aivils Let's wrap this entire operation with a `rescue` so that if there is an exception (e.g. failing network API call) the account is simply logged and skipped, so that the long operation to update all accounts can continue.